noDefaultMetrics.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Prometheus client
+ * @module Prometheus client
+ */
+
+'use strict';
+
+exports.register = require('./lib/register');
+exports.contentType = require('./lib/register').contentType;
+
+exports.Counter = require('./lib/counter');
+exports.Gauge = require('./lib/gauge');
+exports.Histogram = require('./lib/histogram');
+exports.Summary = require('./lib/summary');
+exports.Pushgateway = require('./lib/pushgateway');
+
+exports.linearBuckets = require('./lib/bucketGenerators').linearBuckets;
+exports.exponentialBuckets = require('./lib/bucketGenerators').exponentialBuckets;
```

Comment:
This isn't very DRY – I would suggest updating `index.js` as well and make `index.js` be the exported stuff from this module, but with defaultMetrics added on top of it